src/main/java/frc/robot/subsystems/drivetrain/Drivetrain.java

```diff
@@ -1,10 +1,130 @@
 package frc.robot.subsystems.drivetrain;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.Robot;
+import frc.robot.subsystems.UnitModel;
+
+/**
+ * Build 4 motors, 2 for each side.
+ */
+public class Drivetrain extends SubsystemBase {
+    private final TalonFX masterRight = new TalonFX(Ports.Drivetrain.MASTER_RIGHT);
+    private final TalonFX slaveRight = new TalonFX(Ports.Drivetrain.SLAVE_RIGHT);
+    private final TalonFX masterLeft = new TalonFX(Ports.Drivetrain.MASTER_LEFT);
+    private final TalonFX slaveLeft = new TalonFX(Ports.Drivetrain.SLAVE_LEFT);
+    public UnitModel highGear = new UnitModel(Constants.Drivetrain.TICKS_PER_METER);
+    private UnitModel lowGear = new UnitModel(Constants.Drivetrain.TICKS_PER_METER_2);
+    private Solenoid piston = new Solenoid(Ports.Drivetrain.SOLENOID);
+
+    /**
+     * Invert masterRight motor and set both slaves to follow.
```

Comment:
No need to use camelCase in English.
```suggestion
     * Invert right master motor and set both slaves to follow.
```
But anyway, no need to document this in the constructor.